WebApplication1/Repositories/MovieRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using WebApplication1.Data;
+using WebApplication1.Interfaces;
+using WebApplication1.Models;
+
+namespace WebApplication1.Repositories
+{
+    public class MovieRepository:IMovieRepository
+    {
+        private readonly MovieDbContext _context;
+        public MovieRepository(MovieDbContext context)
+        {
+            _context = context;
+        }
+        public ICollection<Movie> GetMovies()
```

Comment:
this should return a List specifically not ICollection, since this is the data you are going to use

Reply:
same for the others

Reply:
sir, in the repositories, I returned  ().ToList(); Get method. Do I need to change it anyway?